generators/client/templates/vue/.eslintignore.ejs

```diff
@@ -0,0 +1,31 @@
+<%#
```

Comment:
Can you create `generators/client/files-common.js` and move this file to `generators/client/templates/common`?
Seems identical at every framework.

Reply:
did it !